lib/matplotlib/backend_bases.py

```diff
@@ -55,9 +55,12 @@
 
 try:
     from PIL import Image
+    from PIL import PILLOW_VERSION
+    from distutils.version import LooseVersion
+    assert LooseVersion(PILLOW_VERSION) >= LooseVersion("3.4")
     _has_pil = True
     del Image
-except ImportError:
+except (ImportError, AssertionError):
```

Comment:
Instead of catching an `AssertionError`, I think it would be better to have an if block that checks the version.

Reply:
Ok, changed that now. Any reason for this? Is it bad style or does it have any side effects?